config/locales/zxcvbn/en.yml

```diff
@@ -17,6 +17,7 @@ en:
         For a stronger password, use a few words separated by spaces, but avoid
         common phrases
       names_and_surnames_by_themselves_are_easy_to_guess: Names and surnames by themselves are easy to guess
+      no_need_for_symbols_digits_or_uppercase_letters: No need for symbols, digits, or uppercase letters
```

Comment:
we'll need to request spanish and fresh translations of these strings (which is why the specs are failing)

some info here: https://handbook.login.gov/articles/appdev-i18n.html#translating-content-guidance